test/mocha/20-interop-issuer.js

```diff
@@ -0,0 +1,31 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {config} = require('bedrock');
+const {create} = require('apisauce');
+const {httpsAgent} = require('bedrock-https-agent');
+// const sinon = require('sinon');
+// const brPassport = require('bedrock-passport');
+
+const api = create({
+  baseURL: `${config.server.baseUri}/credentials`,
+  httpsAgent,
+  timeout: 1000,
+});
+
+// FIXME: using embedded context:
+// https://www.w3.org/2018/credentials/examples/v1
+// is this context supposed to be supported in a documentLoader
```

Comment:
No, we don't need to include it in the documentLoader... just specify the extra terms via an embedded context in the tests.